MapboxNavigation/CarPlayManager.swift

```diff
@@ -664,6 +663,24 @@ extension CarPlayManager: CPMapTemplateDelegate {
         let shiftedCenterCoordinate = mapView.centerCoordinate.coordinate(at: distance, facing: shiftedDirection)
         mapView.setCenter(shiftedCenterCoordinate, animated: true)
     }
+
+    private func popToRootTemplate(interfaceController: CPInterfaceController?, animated: Bool) {
+        guard let interfaceController = interfaceController else { return }
+        // TODO: uncomment
+//        if #available(iOS 14.0, *) {
```

Comment:
Why this part is commented and marked as `TODO`? What about using `open func popToRootTemplate(animated: Bool, completion: ((Bool, Error?) -> Void)? = nil)` for iOS `14.0` and keep using `open func popToRootTemplate(animated: Bool)` for previous versions?

Reply:
because the xcode <12 don't know about the `open func popToRootTemplate(animated: Bool, completion: ((Bool, Error?) -> Void)? = nil)` and hiding it inside an if statement doesn't mean that it shouldn't know it. I'll enhance the comment

Reply:
ok, you've just done it, never mind